e2e/local-e2e.sh

```diff
@@ -0,0 +1,84 @@
+################################################################################
+#
+# This script is meant to be run from the root of this project
+#
+################################################################################
+
+#!/bin/bash
+
+set -euo pipefail
+
+
+function run() {
+  minikube_up
+  generate_manifests
+  deploy
+}
+
+# Create local minikube cluster and deploys the dev env for parca and parca agent
+function minikube_up(){
+  echo "Spinning up a parca dev cluster"
+  minikube start -p parca-dev \
+    --driver=kvm2 \
+    --kubernetes-version=v1.22.3 \
+    --cpus=4 \
+    --memory=16gb \
+    --disk-size=20gb \
+    --docker-opt dns=8.8.8.8 \
+    --docker-opt default-ulimit=memlock=9223372036854775807:9223372036854775807
+}
+
+#Delete minikube instance
+function minikube_down(){
+  echo "Deleting parca-dev cluster"
+  minikube delete -p parca-dev
+}
+
+# Configure clusters to run latest commit in Parca agent
+function deploy() {
+  SERVER_LATEST_VERSION=$(curl -s https://api.github.com/repos/parca-dev/parca/releases/latest | grep -oP '"tag_name": "\K(.*)(?=")' | xargs echo -n)
+  AGENT_LATEST_VERSION=$(curl -s https://api.github.com/repos/parca-dev/parca-agent/releases/latest | grep -oP '"tag_name": "\K(.*)(?=")' | xargs echo -n)
+
+  echo "Server version: $SERVER_LATEST_VERSION"
+  echo "Agent version: $AGENT_LATEST_VERSION"
+
+  kubectl apply -f https://github.com/parca-dev/parca/releases/download/"$SERVER_LATEST_VERSION"/kubernetes-manifest.yaml
+  kubectl apply -f ./manifests/local/manifest-e2e.yaml
+
+  sleep 1m  #add a probe or something
```

Comment:
You can check using kubectl if they are rollout.

https://kubernetes.io/docs/reference/kubectl/cheatsheet/#updating-resources
https://jamesdefabia.github.io/docs/user-guide/kubectl/kubectl_rollout/
https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#-em-status-em-
https://kubernetes.io/docs/tasks/manage-daemon/update-daemon-set/